client/client.go

```diff
@@ -1096,6 +1156,9 @@ func addrsToUrls(addrs []string) []string {
 
 // IsLeaderChange will determine whether there is a leader change.
 func IsLeaderChange(err error) bool {
+	if err == errs.ErrClientTSOStreamClosed {
```

Comment:
Why judge it?

Reply:
There is specific change for this. When TSO service switched, the stream(TSO_TSOClient or PD_TSOClient)'s send/recv could return io.EOF. In this case, we need to treat it as LeaderChanged and update leader address immediate. Previously, we didn't handle this case, and now this change handles it.